src/actions/suggestions.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+export default function suggestionsReducer (state, action) {
```

Comment:
Do we run a combineReducer() feature at all? Could potentially clarify some things and make implementation in the src/index.js less irksome. I'd prefer this to have a file path like `src/reducers/index.js`, then import the smaller reducers into and run a combineReducer(). If we're not open sourcing it, it won't matter, but a suggestion for clarity's' sake.